### PR TITLE
Mxs 1

### PR DESCRIPTION
**Pull request by lizbrinz**

### CODE REVIEW

**Comment by lizbrinz** [comment]
Merged to Master

File: screenlog.0 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[K[maxscale@localhost MaxScale]$ ls
```
Comment (markus456) [inline screenlog.0:1]: This shouldn't be a part of the commit.

File: server/modules/filter/luafilter.c [hunk 2]
```diff
@@ -105,7 +105,7 @@ static FILTER_OBJECT MyObject =
  * The module entry point routine. It is this routine that
  * must populate the structure that is referred to as the
  * "module object", this is a structure with the set of
- * external entry points for this module.
+ * external entry ponits for this module.
```
Comment (markus456) [inline server/modules/filter/luafilter.c:108]: Although fixing typos is a good thing, they should be done as a separate change in a different pull request if they are done. I think we can leave the typos in other modules be for now.

**Comment by markus456** [comment]
This is a good start but usually pull requests are done once the code is complete. I've created the MXS-1 branch in the MaxScale repo where you can do the pull requests once the coding progresses. You can close this pull request and do a new one against the MXS-1 branch once you have finished implementing the filter skeleton.

Keep up the good work!
